app/components/arclight/search_result_component.html.erb

```diff
@@ -43,5 +38,9 @@
         <% end %>
       </div>
     </div>
-  </div>
+
+
+    <div class="col-sm-2">
+      <%= helpers.render_index_doc_actions document, wrapping_class: 'd-inline-flex justify-content-end' %>
```

Comment:
If the intent is for the document action controls to align right, I think the bootstrap flex/content alignment classes should be on the column div? I'm not sure if that's what's intended or not though.

As written:
<img width="997" alt="Screen Shot 2022-11-08 at 9 41 12 AM" src="https://user-images.githubusercontent.com/458247/200594463-4535c750-5b33-4874-a373-5a66bfb50914.png">

My proposed change:
<img width="1005" alt="Screen Shot 2022-11-08 at 9 41 43 AM" src="https://user-images.githubusercontent.com/458247/200594522-1333854e-0ae8-4e72-9768-55ebb5e07b3c.png">

```suggestion
    <div class="col-sm-2 d-inline-flex justify-content-end">
      <%= helpers.render_index_doc_actions document %>
```

Reply:
No, the intention is not for them to align right, but to align left to the grid at +10.